Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml

```diff
@@ -407,8 +407,9 @@ script:
     deprecated: false
     description: Searches for sessions in AutoFocus. To view results, run the autofocus-sessions-search-results
       command with the returned AF Cookie. The AF Cookie expires 120 seconds after
-      the search completes. `Autofocus Query Samples, Sessions and Tags` Playbook
-      is recommended for querying and polling.
+      the search completes. Note that if you pass more than 100 IPs, it may not work when searching the result with the returned AF cookie,
+      due to an API limitation. In this case, you will need to iterate over the IPs and searching the result for each AF cookie.
```

Comment:
1. what do you mean by "searching the result"? maybe just "searching"?
2. is it only for IPs or also for other IOCs?